docs/source/science/OPERA/OPERA_Surface_Displacement.ipynb

```diff
@@ -0,0 +1,5049 @@
+{
```

Comment:
<p>Why did we do this instead of the spatial subsetting example? Since it's single timestamp, I think a subset of variables in the X,Y plotted should demo what we want. Perhaps a histogram of values. I suppose mean could be interesting but 'displacement' is the variable that's most interesting and easiest to understand.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>